setup.py

```diff
@@ -28,27 +27,35 @@
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
         "Framework :: Django",
         "Framework :: Django :: 3.1",
         "Framework :: Django :: 3.2",
+        "Framework :: Django :: 4.0",
         "Framework :: Wagtail",
         "Framework :: Wagtail :: 2",
+        "Framework :: Wagtail :: 3",
     ],
     install_requires=[
-        "Django>=3.1,<3.3",
-        "Wagtail>=2.14,<2.17",
+        "Django>=3.1,<4.1",
+        "Wagtail>=2.14,<4.0",
         "lxml>=4.7,<4.8",
         "bleach>=4.1,<4.2",
         "prettytable>=2.2,<2.3",
         "shortcodes>=5.1,<6.0",
+        "cached-property>=1.5,<2.0",
```

Comment:
I get an error when installing this revision with Poetry
`Because wagtail-wordpress-import (rev wagtail-3.0) depends on cached-property (>=1.5,<2.0) which doesn't match any versions, wagtail-wordpress-import is forbidden.`

https://pypi.org/project/cached-property/#history

Looking at releases history, it looks like `1.5` does not exist, and it might be the cause of this issue. 
Maybe we could set the constraint as follows ?
```suggestion
        "cached-property>=1.5.1,<2.0",
```